client/.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
The ember-cli generated it, I don't think there is an issue with bumping it.

Nodes versioning scheme is bit screwy at the moment 0.12 and 4.x I think are somewhat close and 5.x is the latest. They went through some merging of a fork into the mainline and overhaul of version numbers as. I would assume the travis-ci will have no issues with 5.x series